daemon/stop.go

```diff
@@ -38,9 +38,10 @@ func (daemon *Daemon) containerStop(container *container.Container, seconds int)
 		return nil
 	}
 
+	stopSignal := container.StopSignal()
 	// 1. Send a SIGTERM
-	if err := daemon.killPossiblyDeadProcess(container, container.StopSignal()); err != nil {
-		logrus.Infof("Failed to send SIGTERM to the process, force killing")
+	if err := daemon.killPossiblyDeadProcess(container, stopSignal); err != nil {
+		logrus.Infof("Failed to send signal %d to the process, force killing", stopSignal)
```

Comment:
Alright; was thinking to use a named signal here, because the "old" version also used that. But I'm ok with using the number

@vdemeester wdyt?